docs/source/api/cdns_name_configs_routing.rst

```diff
@@ -18,7 +18,8 @@
 *********************************
 ``cdns/{{name}}/configs/routing``
 *********************************
-.. caution:: This API route is currently broken, see :issue:`2941` for more information.
+.. deprecated:: 1.5
```

Comment:
we are using ATC version for deprecation notices so this should be `ATCv4`

Reply:
Changed in 80378e6d8d.